avm/ptn/security/security-center/README.md

```diff
@@ -106,12 +106,22 @@ module securityCenter 'br/public:avm/ptn/security/security-center:<version>' = {
     deviceSecurityGroupProperties: {}
     ioTSecuritySolutionProperties: {}
     location: '<location>'
-    securityContactProperties: {
-      alertNotifications: 'Off'
-      alertsToAdmins: 'Off'
-      email: 'foo@contoso.com'
+    securityContactsProperties: {
+      alertMinimalSeverity: 'Low'
+      attackMinimalRiskLevel: 'Low'
+      emails: 'foo@contoso.com'
+      isEnabled: true
+      notificationsByRole: {
+        roles: [
+          'AccountAdmin'
```

Comment:
Is this classic co-admin role? If yes then 'Owner' might be more suitable as an example.

Ref: https://learn.microsoft.com/en-us/azure/role-based-access-control/classic-administrators?tabs=azure-portal#frequently-asked-questions